README.md

```diff
@@ -140,6 +140,7 @@ Note that installing from OS packages from [the download page](https://downloads
 That requires [bundler](http://bundler.io/):
 
 ```bash
+# From ./inspec-bin directory
 bundle install
 bundle exec bin/inspec help
```

Comment:
I agree the instructions are wrong, but I don't think we should refer to inspec-bin at all - it isn't necessary. Instead change the invocation to:
`bundle exec inspec help` (from the source root) and that has always been how I have ran it.

Reply:
Thanks much! 👍